lib/index.d.ts

```diff
@@ -504,6 +504,16 @@ declare namespace expect {
          */
         above(value: T): Assertion<T>;
 
+        /**
+         * Asserts that the reference value is within a delta difference from the provided value.
+         * 
+         * @param value - the value to compare to.
+         * @param delta - the delta +/- range value.
+         *
+         * @returns assertion chain object.
+         */
+        about(value: T, dela: number): Assertion<T>;
```

Comment:
Indeed, I didn't notice `above` was using the same pattern. You're right, for now the goal is to keep types as is per the previous discussion in https://github.com/hapijs/hapi/issues/4388 and we'll iterate later on to improve upon them. We can keep it as is for now.

Reply:
If you look at the [matching logic](https://github.com/hapijs/code/blob/aa1f2b36bcdd56d69f876af5d1e2b114807c80d2/lib/index.d.ts#L98-L103), it is actually `number | bigint`.

Ideally it should only match the tested type, since you can't mix the `number` and `bigint` types.

Reply:
Thanks @kanongil for the explanation. It seems my suggestion was wrong then. The way I see it we should keep as it is but do you wish to make a change here or may I proceed with merging and publishing it?